plugin/trino-memsql/src/test/java/io/trino/plugin/memsql/TestMemSqlTypeMapping.java

```diff
@@ -165,37 +148,25 @@ public void testFloat()
 
     @Test
     public void testDouble()
-    {
-        doublePrecisionFloatingPointTests(doubleDataType())
-                .execute(getQueryRunner(), trinoCreateAsSelect("trino_test_double"));
-        doublePrecisionFloatingPointTests(memSqlDoubleDataType())
-                .execute(getQueryRunner(), memSqlCreateAndInsert("tpch.memsql_test_double"));
-    }
-
-    private static DataTypeTest doublePrecisionFloatingPointTests(DataType<Double> doubleType)
     {
         // we are not testing Nan/-Infinity/+Infinity as those are not supported by MemSQL
```

Comment:
Can we retest this assumption to see if it still holds? We updated the testing MemSQL version in the past.

Reply:
The sentence is still true.